config/initializers/devise.rb

```diff
@@ -0,0 +1,299 @@
+# frozen_string_literal: true
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '4bbf260a69902be26b59e538e37891fcd1eda9f34813593dd3090f9d16c705fde60c695fa32fb9313d41d7584e325cb8c68faae6b95a1ec8b3a84e8e32c87cfa'
```

Comment:
ensure this is stored in ENV / secrets

Reply:
Good idea - just set this up in 48ad315197bf0e5a276bcf39db10aaae3a647b3d.